package.json

```diff
@@ -38,13 +38,16 @@
     "marked": "^0.5.2",
     "natives": "^1.1.6",
     "node-fetch": "^2.6.0",
-    "prismic-javascript": "^2.7.1",
     "prismjs": "^1.16.0",
     "rollup-plugin-dotenv": "^0.3.0",
     "stencil-router-v2": "^0.5.0",
     "typedoc": "^0.17.7",
     "workbox-build": "4.3.1"
   },
+  "dependencies": {
```

Comment:
Can these be under dev-dependencies? Technically, there's a lot that should be moved to "dependencies", but it doesn't make sense to have these two be the only ones.

Reply:
I thought dependencies are the ones that are required at runtime?
https://github.com/ionic-team/capacitor-site/blob/add-prismic-ads/src/components/internal-ad/internal-ad.tsx#L58

Reply:
Well looks to still work when under dev dependencies